### ISSUE TITLE
Attributes not working.

### ISSUE THREAD

**Issue by Tjoosten** [spatie/laravel-route-attributes#7]
Hey Spatie, 

Currently im trying this excellent package. On a fresh application. But i keep on stumbling on that the attribute is not working and always returning a 404 response. 

My code: 

```
<?php

namespace App\Http\Controllers;

use Spatie\RouteAttributes\Attributes\Get;

class HomeController extends Controller
{
    #[Get('home' name: "home")]
    public function index()
    {
        return view('home');
    }
}
```

Is this possible a bug?

**Comment by freekmurze** [comment]
According to the tests the package is working as intended. Please submit a failing test for me to fix.

**Comment by chessPlayerDeveloper** [comment]
Hi @freekmurze,

This is the currently RouteAttributeServiceProvider:

```php
protected function registerRoutes(): void
{
    if (! config('route-attributes.enabled')) {
        return;
    }

    $routeRegistrar = new RouteRegistrar(app()->router);

    collect(config('directories'))->each(fn (string $directory) => $routeRegistrar->registerDirectory($directory));
}
```

Maybe must be?

```php
protected function registerRoutes(): void
{
    if (! config('route-attributes.enabled')) {
        return;
    }

    $routeRegistrar = (new RouteRegistrar(app()->router))
        ->useRootNamespace('App\\');

    collect(config('route-attributes.directories'))->each(fn (string $directory) => $routeRegistrar->registerDirectory($directory));
}
```

The string rootNamespace is not defined and config directories not exists.

With this modification all works fine.

Thanks.

**Comment by HepplerDotNet** [comment]
That also fixed my issue #12
